documentation/tests/test_documentation_rendering.py

```diff
@@ -0,0 +1,30 @@
+from django.test import TestCase
+
+from curricula.factories import CurriculumFactory, UnitFactory
+from lessons.factories import LessonFactory
+
+from documentation.factories import IDEFactory, BlockFactory
+
+
+class DocumentationRenderingTestCase(TestCase):
+    def setUp(self):
+        self.myIDE = IDEFactory(slug="myIDE")
+        self.myBlock = BlockFactory(slug="myBlock", title="My Block", parent_ide=self.myIDE, parent=self.myIDE)
+
+        self.test_curriculum = CurriculumFactory(slug="test-curriculum")
+        self.test_unit = UnitFactory(parent=self.test_curriculum, slug="test-unit")
+
+        self.test_lesson = LessonFactory(title="My Lesson", parent=self.test_unit)
+        self.test_lesson.blocks.add(self.myBlock)
+
+    def test_render_course_blocks_page(self):
+        response = self.client.get('/test-curriculum/code/')
```

Comment:
what is the significance of `code` in this URL?

Reply:
nm, found it -- was looking in documentation urls instead of curricula urls. https://github.com/code-dot-org/curriculumbuilder/blob/e2a7c0ef6389791bd0558993652f4109c0bbd180/curricula/urls.py#L31-L31